lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -120,6 +120,15 @@ function dosomething_helpers_extract_field_data($field, $language = LANGUAGE_NON
 
     $data = $field[$language];
 
+    if (!isset($data)) {
+      if (isset($field[LANGUAGE_NONE])) {
+        $data = $field[LANGUAGE_NONE];
+      }
+      else {
+        return;
```

Comment:
I guess it could return an error, but the point of returning nothing was there is literally nothing to return.

if you specify a language and there is no content for it OR LANGUAGE_NONE, than there isnt anything to return thats relevant to what you requested.

actually, can we just make it return NULL?